chain/network/src/utils.rs

```diff
@@ -5,6 +5,12 @@ use cached::SizedCache;
 
 use crate::types::{BlockedPorts, PatternAddr};
 
+// `blacklist_from_iter` - converts list of addresses represented by strings to <IpAddr, BlockedPorts> HashMap
```

Comment:
nit: you might want to use '///' -- this way this is marked as 'documentation' rather than just a comment (and some of the rust tools can pick it up)